src/browser/atom-application.coffee

```diff
@@ -491,7 +491,7 @@ class AtomApplication
   #   :specPath - The directory to load specs from.
   #   :safeMode - A Boolean that, if true, won't run specs from ~/.atom/packages
   #               and ~/.atom/dev/packages, defaults to false.
-  runSpecs: ({exitWhenDone, resourcePath, specDirectory, logFile, safeMode}) ->
+  runSpecs: ({exitWhenDone, resourcePath, specDirectory, logFile, safeMode, testRunnerBootstrap}) ->
```

Comment:
:memo: Would be great to doc `testRunnerBootstrap` in the comment for this method.